### ISSUE TITLE
README fails to mention website blocks for "Gaming Full" template

### ISSUE THREAD

**Issue by DenebTM** [KodoPengin/GameIndustry-hosts-Template#43]
If you ask me, a blocklist specifically aimed at eliminating tracking in games should not directly interfere with general web surfing. I understand that some games may connect to e.g. `[www.]facebook.com` or `www.instagram.com` directly, necessitating a block — but the README should make it clear that certain websites will be blocked, and perhaps they should be mentioned by name at the top of the list itself.

**Side note:** OpenWRT's "adblock" package features the _Main_ template as an included option, calling it `games_tracking`. I have raised an [issue](https://github.com/openwrt/packages/issues/24881) with them a while ago, where so far the only response has been "not our problem, report upstream". That's not your problem imho, I just thought I'd mention it.

**Side note Nr. 2:** Though I get the sentiment, I wouldn't expect a tracking-focused blocklist to totally block access to specific "news" websites on account of being unreliable or sensationalist. I would suggest making this more obvious in the description as well.

**Comment by RandomGuy28** [comment]
Agreed, the **Gaming Mini** template was perfect but it's gone now and I tried the **Gaming Full** template but had to stop using it because it blocks a lot of irrelevant things.

**Comment by KodoPengin** [comment]
Hello Deneb,

It's mentioned on the main page:

- Privacy-infringing tech companies like Facebook are blocked.
- A significant number of outgoing connections from Windows (xx) and Microsoft games is blocked.
- Strict policy with no exceptions for large corporations.

The problem with Facebook in the example is above all that trackers converge under the main domain and are therefore naturally also blocked. This is important for desktop applications as well as and especially for Android systems.

In this sense, the list does exactly what it is supposed to do. If you want to use convenience functions, such as Facebook services in this case, these must be activated manually.

**Comment by DenebTM** [comment]
> Out of curiosity, can I ask which websites you have in mind? And what makes you think that they are blocked because of their content rather than their tracking behavior? 🤔

I've mentioned two: `bild.de` and 

Also, please do get out of your sensationalist right-wing media bubble.
> 
> Because I know for a fact that certain ad-block lists, like ROad-Block, and hosts lists, like Steven Black's fakenews, deliberately block independent media that reports critically on US involvement Syria; the Georgian parliamentary crisis; the suspension of democracy in the UK, France, Germany, Romania and South Korea; the Ukrainian invasion of Russia; the Covid-19 pandemic and vaccines; Extinction Rebellion, BLM, KKK, ADL, SPLC and Antifa; American and Brazilian election fraud; the US border crisis; immigration and crime statistics; illegal abortion clinics; surgeons committing illicit genital mutilation and hormone poisoning against minors, etc. But I've never noticed anything like that from Kodo Pengin's hosts lists.
> 
> What I have run into though, is that I'm unable to visit a whole bunch of "legitimate" websites (e.g. codeberg.org, distrowatch.com, speedtest.net, whatever.social and ggtyler.dev), after adding versions of the three extensions (Gaming-Full, Android-Mini, Amazon-FireStick) from the past year to my hosts file. One or more of them seem to be blocking some Certificate Issuing Authorities or OCSP responder servers, thereby causing OCSP errors.
> 
> I've been meaning to file a bug report about that for a while, but wanted to narrow down which rules are causing the issue first. Then I never got around to doing that, and so here we are now, almost a year later. 😅

**Comment by DenebTM** [comment]
(accidentally closed, oops)

@incrediblyimpressiveusername
> Out of curiosity, can I ask which websites you have in mind? And what makes you think that they are blocked because of their content rather than their tracking behavior? 🤔

I've mentioned two in the linked OpenWRT issue: `bild.de` and `taz.de`.

Also, this is absolutely not the place for you to list all of your personal convictions, please take that somewhere else.

**Comment by DenebTM** [comment]
@KodoPengin
Yeah, would be nice, agreed. Your argument that "making exceptions for your own convenience is contradictory on principle alone" does make sense, but not everyone operates strictly on principle; I don't think it invalid to *cut down* the number of trackers without completely breaking half the web (because half the web is trackers). As @RandomGuy28 alluded to, the "Gaming Mini" was a nice middle-ground, as it offered a choice of "minimise interference outside of games" as opposed to "block everything, regardless what else breaks". Why was it deleted anyway? I couldn't find a reason given.

**Comment by KodoPengin** [comment]
@DenebTM
Bild > Axel Springer > See Adblock Discussion since many years as well as for excessive tracker use. Can be read retrospectively, especially in various german articles, and the topic is unfortunately still topic.
TAZ > For TAZ it is a little more. Added because of spreading misinformation and for excessive tracker use  (Well, same counts often for bild). In addition, various trackers are loaded at TAZ, which become active before the consent of the consent manager, as well as other trackers that become active despite rejection

This does not really have anything to do with subjective impressions, but with the fact that certain business practices and criminalization of users are not supported.

@incrediblyimpressiveusername
Actually, only Norton and part of Let's Encrypt (also noted in the hosts file) are blocked from OCSP servers. Everything else works fine. Norton, in terms of credibility (unsafe certs, data security concerns, security gaps to name a few), and Let's Encrypt because they shared a backend with Adobe. Keyword: telemetry service callbacks Adobe AIR.

The other mentioned sites should not cause any issues. I used Codeberg for a long time myself, so that would have conflicted.

--

Overall it's an one-man project.
I maintain all the lists manually. Each entry, along with the raw data, is individually checked and entered into the respective hosts. This takes up a lot of time, and if that’s not enough, maintaining a few lists alone is a 24/7 job. Apart from 2 exceptions (it was about Ebay entries and FBI scam sites), only own researched data is used and no third-party stuff is used.

Additionally, publishing on multiple platforms and responding to requests. Unpaid, without and without support from others. Except for a few feedback responses, which are are very marginal. 

To keep it short: The gaming mini-template was simply outdated. 

It also states in various places that, as previously mentioned the lists are sometimes very strict. Additionally, users have the option to modify entries themselves (if they choose to) or selectively choose entries as needed.

I cannot and will not please everyone, especially because privacy/security (in any form) and convenience or browser habits often exclude each other.
